README.md

```diff
@@ -31,13 +34,23 @@ cargo run --release -- import \
   --snapshot 70070379.d6fe6439aed8bddc10eec22c1575bf0648e4a76125387d9e985e9a3f8342870d.cbor
 ```
 
-2. Setup observability backends:
+2. Import chain data from remote peer
```

Comment:
I would assume this import is temporary-ish, as we probably would came up with a single snapshot format containing everything?
Would this data be available in the `cardano-node` data folder that is signed by `mithril`?

Reply:
While this particular import is definitely temporary, I am not sure we don't want to have ways to separate importing the raw chain date from the ledger. Mithril bundles it all in the cardano-node db snapshot, but anyhow the ledger snapshot bundled there is not suitable for use by Amaru for now.